src/components/ItemsList.js

```diff
@@ -43,41 +47,65 @@ const ItemsList = () => {
     history.push('/additem');
   };
 
+  const clearText = (event) => {
+    event.preventDefault();
+    setSearchTerm('');
+  };
 
   return (
     <div className="items-list">
       <h1>Your Shopping List</h1>
       {loading && <p>Loading...</p>}
       {error && <p>An error has occured...</p>}
       {shoppingList && !shoppingList.length && (
-            <div className="add-item">
-              <p>You haven't created a shopping list yet...</p>
-              <button type="submit" onClick={handleRedirect}>
-                Add First Item
-              </button>
-            </div>
-          )}
-      <form>
+        <div className="add-item">
+          <p>You haven't created a shopping list yet...</p>
+          <button type="submit" onClick={handleRedirect}>
+            Add First Item
+          </button>
+        </div>
+      )}
+      <form onReset={clearText} className="search-list">
+        {shoppingList && shoppingList[0] ? (
+          <div>
+            <input
+              type="text"
+              placeholder="Search list.."
+              value={searchTerm}
+              onChange={handleChange}
+            />
+            <input type="reset" value="X" />
+          </div>
+        ) : null}
```

Comment:
I was just re-reading the AC's and maybe it is not very clear there but I think the idea is to conditionally render "X" button if the text is entered in the search bar. 
AC: `When the field has text in it, the user should be able to tap a UI element (e.g., with an "X" button next to the field) to clear the field`.

To achieve this functionality, I think it would be a good idea to have a `<button>` tag rather than an `<input>` with reset type.